README.md

```diff
@@ -18,8 +18,8 @@ This repository exists to store large binaries and similar artifacts needed as t
 To add new test data do the following:
 
 1. Create a new release in the repository using the GitHub interface.
-2. Set the title to be "Dataset {major}.{minor}" (for example "Dataset 3.0"). Most datasets will only need a major version number, but if you need to upload a multi-part dataset use the minor version. When in doubt, update the major version.
-3. Set the tag to "dataset-{major}.{minor}" (for example "dataset-3.0").
+2. Set the title using the format "Dataset YYYY-MM-DD" (for example "Dataset 2025-02-19").
+3. Set the tag to "dataset-YYYY-MM-DD" (for example "dataset-2025-02-19").
```

Comment:
Please add something of the form: if you ever need to add a second tag in one day, use `dataset-2025-02-19b` (or whatever suffixing you think is best, be it `b` or `-2` or `-oops` or whatever) :)